SeleniumSolarTechnology/SolarTechnology/Tests/UITestFixture.cs

```diff
@@ -0,0 +1,36 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium.Support.UI;
+
+namespace SolarTechnology.Tests
+{
+    public class UITestFixture
+    {
+        [SetUp]
+        public void Setup()
+        {
+            var options = new ChromeOptions();
+            options.AddArgument("window-size=1400,1200"); // Set desired resolution
+            _driver = new ChromeDriver(options);
+            _driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
```

Comment:
since Explicit waits are used in the project, this ImplicitWait should be removed

Reply:
Clearly, but when i removed this ImplicitWait  the Test VerifyAddToCartAndRemoveFromCartProductItem failed...

Reply:
Then it means that that test requires some additional explicit wait )